File: tooling/sdkgen/test/dotnet/UnityTestbed/Assets/Example.Chat.Sdk/Core/Models/Server/SendMessagePayload.cs
using Newtonsoft.Json;

namespace Example.Chat.Core.Models.Server
{
    public partial class SendMessagePayload
    {
        [JsonProperty("userId")]
        public string UserId;

        [JsonProperty("roomId")]
        public string RoomId;

        [JsonProperty("text")]
        public string Text;

    }
}

File: tooling/sdkgen/test/typescript/openws-sdkgen-typescript-node/src/core/models/server/send-message-payload.ts
export interface SendMessagePayloadInit {
    userId: string
    roomId: string
    text: string
}

export class SendMessagePayload implements SendMessagePayloadInit {
    readonly userId: string
    readonly roomId: string
    readonly text: string

    constructor({ userId, roomId, text }: SendMessagePayloadInit = {} as SendMessagePayloadInit) {
        this.userId = userId
        this.roomId = roomId
        this.text = text
    }
}
